### DISCUSSION TITLE
RxJS 7.5.7 => 7.6.0 creates typescript error for Intl.DisplayNames

### DISCUSSION THREAD

**Discussion by troywweber7** [ReactiveX/rxjs#7150] [Q&A]
I wasn't sure this should be reported as a core bug or not, but essentially I'm using Pnpm and Typescript 4.8.4 and one of my files accesses `Intl.DisplayNames`. On RxJS `7.5.7`, there are not errors, but as soon as I upgrade to `7.6.0`, a Typescript error shows up that says `DisplayNames` does not exist on `Intl`.

I'm at a loss for why might have changed that is causing this issue or how I can debug or work around it (short of staying on `7.5.7` for now). It seems like it should be completely unrelated to `RxJS`, yet it is isolated to when I jump the indicated version boundary.

Can anyone provide me some guidance or insight?

**Comment by kwonoj** [comment]
I don't have a good answer. The changes between those versions are small enough - https://github.com/reactivex/rxjs/compare/7.5.7...7.6.0 and I'm not sure what could be a culprit.

One of the possible type changes is https://github.com/reactivex/rxjs/compare/7.5.7...7.6.0#diff-096dc41cab274b2c01f42f58353ac0903b8572c52c3458b5b1461af623fdc5d7R1. but it actually removes dependency to node.js specific types so not sure if that's really related.
